xbmc/interfaces/json-rpc/AudioLibrary.cpp

```diff
@@ -1022,7 +1022,8 @@ JSONRPC_STATUS CAudioLibrary::SetSongDetails(const std::string &method, ITranspo
   if (!musicdatabase.UpdateSong(song, updateartists))
     return InternalError;
 
-  CJSONRPCUtils::NotifyItemUpdated();
+  const auto pItem = std::make_shared<CFileItem>(song);
```

Comment:
Please do not use type prefix for variable's names. Coding guidelines. https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#88-variables

Reply:
Done